AENS.md

```diff
@@ -238,6 +271,98 @@ Note that `expire` is not an explicit message that is part
 of the protocol.
 
 
+### Protocol fees and protection times
+
+Here is the function of initial bidding price depending on name length for Lima hardfork.
+
+```
+ ------------- -------------
```

Comment:
Needs an update?

Reply:
@hanssv No, this is synced with PR to full node. Maybe you mean something else?